### ISSUE TITLE
Explore Interaction Pattern for Listboxes and Action Toolbar in Build an Atom

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#44]
Here's a possible interaction pattern playing with particles in My Atom's regions. The three regions and the particle buckets are marked up as 4 listboxes (see current Parallel Dom for the HTML bing discussed in issue #43).

The  user would use available action buttons for taking actions on selected listbox items (e.g., grab, select positions in regions, move within a region, and delete).

We could customize to simplify, using some of the strategies already mentioned, however,
I'd like to see how it works with a purely native approach.

Should I copy what I have from the design document [section Idea 2 Interaction Pattern](https://docs.google.com/document/d/1QzdjwByRXAXKAOLpv-3VPUihu_IuTSZ4pmvHFt6mn98/edit#heading=h.gcnwb2jsi26y)? Or just link to the design document?

@zepumph , @jessegreenberg , @emily-phet let me know what you think about these interaction patterns.

**Comment by emily-phet** [comment]
@terracoda 
I think just linking to the doc should be fine (since it's lengthy).

I think I understand the interaction (it's a bit difficult for me, since I'm unfamiliar with navigating listboxes without a mouse). 
Can you describe what a visual user would experience? I'm unsure now that I think I understand your design better how this approach will span visual and nonvisual use. (It may be obvious to those more familiar with listboxes, though).

**Comment by terracoda** [comment]
@emily-phet, I am trying to sketch it out visually in a [google slide](https://docs.google.com/presentation/d/1Qta-iRfy_GrW_6IRuXeKagKvBIzQ5ipUqK8pMPgmT9g/edit#slide=id.g1f882a10cf_0_69), but it's a bit slow. I've got to slide 5 today.

**Comment by terracoda** [comment]
@emily-phet, I am trying to see how we can map it pretty close to the visual design strategy we have already developed, but maybe with a some slightly updated highlighting and of course the key presses are different.

**Comment by zepumph** [comment]
I looked through the document today, and although I didn't understand it completely. I like the approach! It is cool to try and solve as much of the interaction and structure as possible natively. I would love to talk more about your vision with the action bar, are you planning on talking about this tomorrow in the IDRC meeting?

**Comment by terracoda** [comment]
@zepumph, yes, I am trying to visually map out the PDOM and visual sim in the google slides for today's IDRC meeting. You are welcome to comment, but keep in mind that at the moment the slides are still very messy.

I'm not opposed to a custom interaction, but am trying to see how far we can get with a native interaction. Also, keeping in mind the benefits of both approaches.

**Comment by zepumph** [comment]
Lowering priority and unassigning until BAA is back on my plate.

**Comment by terracoda** [comment]
@emily-phet, @zepumph I have updated links in the [design document](https://docs.google.com/document/d/1QzdjwByRXAXKAOLpv-3VPUihu_IuTSZ4pmvHFt6mn98/edit) and completed two cognitive walkthroughs in the google slides, [BAA: Listbox Role Cognitive Walkthrough](https://docs.google.com/presentation/d/1Qta-iRfy_GrW_6IRuXeKagKvBIzQ5ipUqK8pMPgmT9g/edit) for when we are ready to return to this sim.
